### PR TITLE
Add a minimumGasLimit to the original gas customization modal

### PR DESCRIPTION
**Pull request by danjm**
An alternative to https://github.com/MetaMask/metamask-extension/pull/9600 that directly updates the existing gas customization modal, as opposed to the new swaps customization modal added in https://github.com/MetaMask/metamask-extension/pull/9599

We will still want to do https://github.com/MetaMask/metamask-extension/pull/9600, but doing it this way allows us to get this fix into prod faster, which will help reduce failed swaps.

### CODE REVIEW

**Review by rekmarks — CHANGES_REQUESTED** [review]
This PR does not account for the usage of the `gasLimitTooLow` locale message as the constant `GAS_LIMIT_TOO_LOW_ERROR_KEY` in `error-keys.js`, which is used to produce error messages in the `ConfirmPageContainerContent` component via the `getErrorKey` method of `ConfirmTransactionBase`.

Perhaps the simplest solution is to preserve the original `gasLimitTooLow` locale message and create a new one that accepts a substitution?

**Review by Gudahtt — CHANGES_REQUESTED** [review]

File: ui/app/components/app/gas-customization/advanced-gas-inputs/advanced-gas-inputs.component.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ export default class AdvancedGasInputs extends Component {
     customPriceIsSafe: PropTypes.bool,
     isSpeedUp: PropTypes.bool,
     customGasLimitMessage: PropTypes.string,
+    minimumGasLimit: PropTypes.number.isRequired,
```
[the remark this answers is not in the document]
Comment (danjm) [reply ui/app/components/app/gas-customization/advanced-gas-inputs/advanced-gas-inputs.component.js]: Addressed in 466e4debb

**Comment by danjm** [comment]
@rekmarks I've addressed your comment in 42cd1091c

**Review by rekmarks — APPROVED** [review]
LGTM!

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Pull request merged**